src/app/search/page.module.css

```
@@ -0,0 +1,49 @@
+.page {
+  .header {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    margin-top: 4rem;
+  }
+  .title {
```

بین Ruleها یک خط فاصله بذارید

```suggestion
  }
  
  .title {
```